contracts/governance/Vesting/VestingRegistryLogic.sol

```diff
@@ -215,6 +228,15 @@ contract VestingRegistryLogic is VestingRegistryStorage {
                 uint256(VestingType.TeamVesting),
                 _vestingCreationType
             );
+
+        vestingCreationAndTypes[vesting] = VestingCreationAndTypeDetail(
+            true,
+            uint120(VestingType.TeamVesting),
+            uint128(_vestingCreationType)
+        );
```

Comment:
it is more clear when assigning named params
```suggestion        
        vestingCreationAndTypes[vesting] = VestingCreationAndTypeDetail(
            {
                isSet: true,
                vestingType: uint120(VestingType.Vesting),
                vestingCreationType: uint128(_vestingCreationType)
            }
        );
```

Reply:
updated in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/447/commits/7f1c269ee40b1d2f3f864b07c574360839a9d270)